src/python/WMCore/ReqMgr/DataStructs/RequestStatus.py

```diff
@@ -124,6 +124,12 @@ def get_modifiable_properties(status = None):
     TODO: Currently gets the result from hardcoded list. change to get from configuration or db
     """
     if status:
-        return ALLOWED_ACTIONS_FOR_STATUS.get(status, 'all')
+        return ALLOWED_ACTIONS_FOR_STATUS.get(status, 'all_attributes')
     else:
         return ALLOWED_ACTIONS_FOR_STATUS
+
+def get_protected_properties():
```

Comment:
Hi Alan, we can add more, these are just auto generated valued a requester shouldn't modify. Maybe I should change the name of the function. If you want me to add more to the list please let me know. We still have time to update today. Thanks